### PR TITLE
Features/confirmed demographics #155196549

### PR DESCRIPTION
**Pull request by fddayan**
https://www.pivotaltracker.com/story/show/155196549

### CODE REVIEW

File: app/javascript/components/molecules/FormGridRow.js [hunk 1]
```diff
@@ -11,4 +11,24 @@ const FormGridRow = ({ label, id, name, placeholder, describeId, note, error })
   )
 }
 
+const FormGrid = {}
```
Comment (callaghanc) [inline app/javascript/components/molecules/FormGridRow.js]: @fddayan Why is this block of code here as well as in the file above?

File: app/javascript/components/supplemental_application/mapProperties.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import _ from 'lodash'
 import moment from 'moment'
 
 import { mapApplication } from '../propMappers'
+import { updateApplicationAction } from './actions'
 
 // NOTE: what kind of saleforce domain object is this one? Fed
```
Comment (callaghanc) [inline app/javascript/components/supplemental_application/mapProperties.js:7]: @fddayan This is a `Field_Update_Comment__c` Salesforce object - if you check out that object in the Workbench SOQL Explorer you can see the fields on it.

File: app/javascript/components/molecules/FormGridRow.js (hunk 1 above)
Comment (fddayan) [reply app/javascript/components/molecules/FormGridRow.js]: forgot to remove it :P

File: app/javascript/components/supplemental_application/mapProperties.js (hunk 2 above)
Comment (fddayan) [reply app/javascript/components/supplemental_application/mapProperties.js:7]: Great thanks.

File: app/javascript/components/Alerts.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+const ERROR_MESSAGE = "Oops! Looks like something went wrong. Please try again."
+
+const info = (message) => alert(message)
+
+const error = () => alert(ERROR_MESSAGE)
```
Comment (callaghanc) [inline app/javascript/components/Alerts.js:5]: @fddayan I added a standard error alert using the same default error message we use on the web app for consistency.

**Comment by callaghanc** [comment]
@fddayan looks good! let's chat quickly tmrw about all the mapping - i want to make sure i'm up to date on the whole story there - and then should be good to go 🙂

File: app/javascript/components/Alerts.js (hunk 3 above)
Comment (fddayan) [reply app/javascript/components/Alerts.js:5]: 👍

**Comment by fddayan** [comment]
@callaghanc fixed

**Review by callaghanc — APPROVED** [review]
👍

**Pull request merged**